charts/illumidesk/templates/grader_setup_service.yaml

```diff
@@ -107,14 +107,23 @@ spec:
           env:
             - name: GRADER_IMAGE_NAME
               value: {{ .Values.graderSetupService.graderImage }}
-            - name: POSTGRES_NBGRADER_HOST
+            - name: POSTGRES_GRADER_SETUP_HOST
               value: {{ .Values.graderSetupService.postgresNBGraderHost }}
-            - name: POSTGRES_NBGRADER_USER
+            - name: POSTGRES_GRADER_SETUP_USER
               value: {{ .Values.graderSetupService.postgresNBGraderUser | default "postgres" }}
-            - name: POSTGRES_NBGRADER_PASSWORD
+            - name: POSTGRES_GRADER_SETUP_PASSWORD
               value: {{ .Values.graderSetupService.postgresNBGraderPassword | default "illumidesk" }}
             - name: ILLUMIDESK_MNT_ROOT
               value: '/illumidesk-courses'
+            - name: PORT
+              value: "8000"
+              # empty string for more logging
+            - name: GIN_MODE
+              value: ""
```

Comment:
It seems that this environment variable is going to be present regardless but does not accept any values from a custom-config.yaml (or equivalent) or the `values.yaml`. Are there any scenarios where this setting would require a value other than an empty string?

Reply:
when it is set to "release", there are fewer logs

Reply:
@Abhi94N ok then this needs to be documented in the README. Also, if you can set it to `release` then the values.yaml should accept a custom value that we can override and default to an empty string. I don't see that implementation.